### ISSUE TITLE
Improve/fix documentation of Modelica.Electrical.Multiphase

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#277]
**Modified by dietmarw on 12 Feb 2010 09:29 UTC**
See attached [raw-attachment:Modelica.Electrical.MultiPhase_StyleCheckLog.html StyleCheckLog.html] for indicators what needs to be improved/fixed.

----

**Reported by dietmarw on 12 Feb 2010 09:28 UTC**
See attached [raw-attachment:Modelica.Electrical.Multiphase_StyleCheckLog.html StyleCheckLog.html] for indicators what needs to be improved/fixed.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/277

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 12 Feb 2010 09:29 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 12 Feb 2010 10:52 UTC**
Fixed (d2c0edd28d0c71adb9c7bfcffe77c1c0804fe03e); didn't fix:
(Possibly) insufficient documentation (it is sufficient).
Name shall not contain '_' (PlugToPin_p, PlugToPin_n) in Basic (backward compatibility).
Mixing of components and equations (Basic.Star, Basic.Delta - what should I fix?):

```
equation
  for j in 1:m loop
    connect(plug_p.pin[j],pin_n);
  end for;
```

```
  for j in 1:m loop
    if j<m then
      connect(plug_n.pin[j],plug_p.pin [j+1]);
    else
      connect(plug_n.pin[j],plug_p.pin [1]);
    end if;
  end for;
```
